modules/core/shared/src/main/scala/lucuma/core/model/SiderealTracking.scala

```diff
@@ -47,9 +47,9 @@ final case class SiderealTracking(
   properMotion:    Option[ProperMotion],
   radialVelocity:  Option[RadialVelocity],
   parallax:        Option[Parallax]
-) {
+) extends Tracking {
 
-  def at(i: Instant): Option[Coordinates] =
+  def apply(i: Instant): Option[Coordinates] =
```

Comment:
this looks odd, I'd expect `apply` for an object to return the same type. would this do
```scala
val a: Option[Coordinates] = SiderealTracking(now)
```

Reply:
Can we have a case class that extends a Function like Tracking? it certainly compiles but I'm wondering about `equals` and `hashCode`

Reply:
> this looks odd, I'd expect `apply` for an object to return the same type. would this do
> 
> ```scala
> val a: Option[Coordinates] = SiderealTracking(now)
> ```

I agree that it would be friendlier to have an explicit name, like `at(Instant)`

Reply:
Or `resolveCoordinatesAt` or whatever, but making it explicit makes the invoking code easier to read.

Reply:
No, it just means you can treat the tracking as a function. It's not a factory method; i.e.,

```scala
val t = SiderealTracking(base, epoch, ...)
val opCoords = t(now)
```